### PR TITLE
Changes to Synchronization API Overview

### PR DESCRIPTION
**Pull request by dhanyahk**
Added inbound flows description to the API overview

### CODE REVIEW

**Comment by dhanyahk** [comment]
@davidmu1  I have incorporated your comments in the previous PR (I had to close it because i messed up the files).

File: concepts/changelog.md [hunk 1]
```diff
@@ -5,12 +5,15 @@ author: "MSGraphDocsVteam"
...
 
-# Changelog for Microsoft Graph
+# Changelog for Microsoft Graph 
 
 This changelog covers specific API-level changes in Microsoft Graph v1.0 and beta.
 
 For a summary of the value of these API changes, as well as recent tools, components, guidance, and tutorial additions, see [What's new in Microsoft Graph](whats-new-overview.md).
 
+## August 2020
+| Change | beta | changed [Syncrhonization API](/graph/api/resources/synchronization-overview?view=graph-rest-1.0) to include inbound provisioning flows.|
```
Comment (davidmu1) [inline concepts/changelog.md:15]: Capitalize "changed".

Also need a header for the change:

## August 2020

### Applications | Synchronization

| Change | beta | Changed [Syncrhonization API](/graph/api/resources/synchronization-overview?view=graph-rest-1.0) to include inbound provisioning flows.|

Where is the actual changed being implemented? I don't see it in this PR.

Comment (dhanyahk) [reply concepts/changelog.md:15]: the intro is the only change. no change to the API itself

Comment (Lauragra) [reply concepts/changelog.md:15]: @dhanyahk , what changed in the introduction? I don't see any changes to the content of the synchronization overview. In any event, we generally don't include updates to conceptual content in the changelog. Can you further clarify the change here? Also note that the name of the API is misspelled in the link text.

Comment (dhanyahk) [reply concepts/changelog.md:15]: we can remove the changelog.md then from the PR. Let me know. As for the changes, here's the updated changes from the original .md file today that's published online. The description has changed and so has the first para. 

--
title: "Azure AD synchronization API overview"
description: "Automate the creation, maintenance, and removal of identities from AD to AAD, HR apps (Workday) to AAD and from AAD to cloud applications."
localization_priority: Normal
doc_type: conceptualPageType
author: "ArvindHarinder1; dhanyahk"
ms.prod: "microsoft-identity-platform"
---

# Azure AD synchronization API overview

Namespace: microsoft.graph

[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]

Azure Active Directory (Azure AD) identity synchronization (also called "provisioning") allows you to automate the provision (creation, maintenance) and de-provision (removal) of identities from:
- Active Directory to Azure Active Directory 
- Workday to Azure Active Directory 
- Azure Active Directory to cloud (software as a service, or SaaS) applications such as Dropbox, Salesforce, ServiceNow, and more. 

You can use the synchronization APIs in Microsoft Graph to manage identity synchronization programmatically, including:

- Create, start, and stop synchronization jobs
- Make changes to the synchronization schema for jobs
- Verify the current synchronization status

File: api-reference/beta/resources/synchronization-overview.md [hunk 2]
```diff
@@ -1,9 +1,9 @@
 ---
 title: "Azure AD synchronization API overview"
-description: "Automate the creation, maintenance, and removal of identities in cloud applications."
+description: "Automate the creation, maintenance, and removal of identities from HR systems, AD, AAD to cloud applications."
```
Comment (Lauragra) [inline api-reference/beta/resources/synchronization-overview.md:3]: Please clarify this description. Where are the identities getting created, maintained, and removed from? The from/to construction is not clear. Please spell out AD and AAD on first mention.

Comment (dhanyahk) [reply api-reference/beta/resources/synchronization-overview.md:3]: I did a detailed one and David wanted me to cut short it.. Will redo it again.

File: api-reference/beta/resources/synchronization-overview.md [hunk 3]
```diff
@@ -13,7 +13,12 @@ Namespace: microsoft.graph
 
 [!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
 
-Azure Active Directory (Azure AD) identity synchronization (also called "provisioning") allows you to automate the creation, maintenance, and removal of identities in cloud (software as a service, or SaaS) applications such as Dropbox, Salesforce, ServiceNow, and more. You can use the synchronization APIs in Microsoft Graph to manage identity synchronization programmatically, including:
+Azure Active Directory (Azure AD) identity synchronization (also called "provisioning") allows you to automate the provision (creation, maintenance) and de-provision (removal) of identities from:
```
Comment (Lauragra) [inline api-reference/beta/resources/synchronization-overview.md:16]: provision -> provisioning, de-provision -> de-provisioning

File: api-reference/beta/resources/synchronization-overview.md (hunk 2 above)
Comment (dhanyahk) [reply api-reference/beta/resources/synchronization-overview.md]: Done. Can you check now? Also expanded on the acronyms

File: api-reference/beta/resources/synchronization-overview.md (hunk 3 above)
Comment (dhanyahk) [reply api-reference/beta/resources/synchronization-overview.md]: Done

File: api-reference/beta/resources/synchronization-overview.md [hunk 4]
```diff
@@ -13,7 +13,12 @@ Namespace: microsoft.graph
...
+- Active Directory to Azure Active Directory.
```
Comment (Lauragra) [inline api-reference/beta/resources/synchronization-overview.md:17]: The from/to statement is not clear. This API is syncing changes from one system to another? Is this list inclusive or is it "one of the following"? Do you mean to say "...allows you to automatically synchronize provisioning and deprovisioning of identities from any of the following:"

File: api-reference/beta/resources/synchronization-overview.md [hunk 5]
```diff
@@ -1,6 +1,6 @@
 ---
 title: "Azure AD synchronization API overview"
-description: "Automate the creation, maintenance, and removal of identities in cloud applications."
+description: "Automate the provisioning of identities from HR systems, Active Directory and Azure Active Directory to cloud applications."
```
Comment (Lauragra) [inline api-reference/beta/resources/synchronization-overview.md:3]: Is "from" the right word? Do you mean the provisioning of identities "in" these systems "to" cloud applications?

File: api-reference/beta/resources/synchronization-overview.md (hunk 4 above)
Comment (ArvindHarinder1) [reply api-reference/beta/resources/synchronization-overview.md:17]: I would change "identities from" to "identities between."

File: api-reference/beta/resources/synchronization-overview.md [hunk 6]
```diff
@@ -13,7 +13,12 @@ Namespace: microsoft.graph
 
 [!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
 
-Azure Active Directory (Azure AD) identity synchronization (also called "provisioning") allows you to automate the creation, maintenance, and removal of identities in cloud (software as a service, or SaaS) applications such as Dropbox, Salesforce, ServiceNow, and more. You can use the synchronization APIs in Microsoft Graph to manage identity synchronization programmatically, including:
+Azure Active Directory (Azure AD) identity synchronization (also called "provisioning") allows you to automate the provision (creation, maintenance) and de-provisioning (removal) of identities from:
+- Active Directory to Azure Active Directory.
+- Workday to Azure Active Directory. 
+- Azure Active Directory to cloud (software as a service, or SaaS) applications such as Dropbox, Salesforce, ServiceNow, and more.
```
Comment (ArvindHarinder1) [inline api-reference/beta/resources/synchronization-overview.md:19]: we can remove the words "cloud (software as a service, or SaaS)"

File: api-reference/beta/resources/synchronization-overview.md (hunk 4 above)
Comment (dhanyahk) [reply api-reference/beta/resources/synchronization-overview.md:17]: It's inclusive.

File: api-reference/beta/resources/synchronization-overview.md (hunk 5 above)
Comment (dhanyahk) [reply api-reference/beta/resources/synchronization-overview.md:3]: We use that interchangeably. So, it's fine

File: api-reference/beta/resources/synchronization-overview.md (hunk 6 above)
Comment (dhanyahk) [reply api-reference/beta/resources/synchronization-overview.md]: Done.

Comment (dhanyahk) [reply api-reference/beta/resources/synchronization-overview.md]: @Lauragra  is there anything else that's open. I fixed all your comments. Please let me know.

File: api-reference/beta/resources/synchronization-overview.md (hunk 4 above)
Comment (dhanyahk) [reply api-reference/beta/resources/synchronization-overview.md:17]: Fixed.

**Pull request merged**